var/lib/delphix-platform/os-migration

```diff
@@ -28,6 +28,19 @@ function die() {
 }
 
 function perform_migration() {
+	#
+	# On Illumos rpool/crashdump has its mountpoint set to "legacy",
+	# and we mount it manually. On Linux we want it to be mounted
```

Comment:
Do you know why we were doing that? My hunch is that the service that runs the savecore code was running before zfs filesystems got mounted, so we needed to mount this manually first (?).

Reply:
We are doing this in `dxinit.sh::do_mount()`. We probably have this set to legacy because `/var` is set to legacy as well and is tied to our Illumos upgrade logic. (i.e. on first boot after upgrade we do a clone of the previous /var dataset in do_datasets() and then mount it in do_mount())